pages/packages/mpi_x/stk/stk_doxygen.md

```diff
@@ -3,13 +3,13 @@ title: STK Doxygen
 permalink: stk_doxygen.html
 folder: mpi_x
 show_sidebar: true
-contact: mwglass@sandia.gov
-package: shards
+contact: william@sandia.gov
```

Comment:
Any reason for why this one doesn't have `STK-NGPTeam@sandia.gov` as contact?

Reply:
No reason. I'll push up a commit to fix that.